recipes/scalene/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set package_name = "scalene" %}
+{% set version = "1.5.10" %}
+
+package:
+  name: {{ package_name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ package_name[0] }}/{{ package_name }}/{{ package_name }}-{{ version }}.tar.gz
+  # See https://pypi.org/project/scalene/#files
+  sha256: 8e2e2ae14acdca56d5bbf33e158814702410f0a72d4e690fce837b811ffadbfc
+
+build:
+  skip: true  # [py<38]
+  skip: true  # [win]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - python
+    - pip
+    - setuptools_scm
+    - wheel
+  run:
+    - python
+    - rich >=9.2.0
+    - cloudpickle >=1.5.0
+    - numpy
+    - pynvml >=11.0.0
+
+test:
+  imports:
+    - {{ package_name }}
+  source_files:
+    - test/original/bm_mdp.py
+  requires:
+    - pyperf
+    - pip
+  commands:
+    - pip check
+    - python3 -m scalene --cli test/original/bm_mdp.py  # [not win]
```

Comment:
```suggestion
    - {{ PYTHON }} -m scalene --cli test/original/bm_mdp.py  # [not win]
```

Reply:
The thing is, it is working but with which Python? If that is failing with our Python it is not working then. Changes are that calling `python3` you are not calling the conda-forge Python on CIs.

Reply:
OK, let's see what happens!